### DISCUSSION TITLE
Calculation interrupt

### DISCUSSION THREAD

**Discussion by Salma-Mao** [idaholab/moose#21263] [Q&A General] [answered]
Hello everyone,
       
When a case reaches 12000 steps, the computer is suddenly restarted and interrupted. How to calculate from 12000 steps？

Thanks,
MS

**Comment by GiudGiud** [comment, marked as answer]
Hello

If it s the computer it has nothing to do with us.
If it s the simulation then please paste the Executioner block here so we can take a look

Guillaume

**Reply by Salma-Mao** [reply]
Hello,

It is computer. I want to  if there is a way to make this case continue to calculate, because recalculation takes a lot of time.

Thanks,
MS

**Reply by GiudGiud** [reply]
they you may use the checkpoint system to restart from where the computer shut down:
https://mooseframework.inl.gov/source/outputs/Checkpoint.html
https://mooseframework.inl.gov/application_usage/restart_recover.html

**Reply by Salma-Mao** [reply]
Thanks for your help.

MS
